### PR TITLE
Add style end fix linters errors

### PR DESCRIPTION
**Pull request by digia3d**
This is the final part of the project. I have done the following:

- Add a progress bar.
- Fix the functionality of the check status button.
- Add style to App to match Zeplin's design.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
 import React from 'react';
+import { FaUserAlt } from 'react-icons/fa';
 
 import { Link } from 'react-router-dom';
 
 function Header() {
```
Comment (FrederickMih) [inline src/components/Header.js:6]: > Hi @digia3d, let's make sure our stylings are EXACTLY the same as that provided on Zeplin. 
> - Kindly align the logo(Bookstore CMS) with the` main-container` as on the Zeplin screenshot below.
> - Once more, let's make sure that the user icon is equally aligned with main-container.Check out the annotated screenshot below.


| Yours     | Zeplin |
| :---:       |    :----:   | 
| annotely_image (11)      | zeplin      |

File: src/components/Book.js [hunk 2]
```diff
@@ -11,23 +12,33 @@ const Book = ({
...
-        <span>Completed</span>
+        <CircularProgress />
+        <div>
+          <p className="text-center">{`${progress}%`}</p>
+          <span className="progress-bar">Completed</span>
+        </div>
       </div>
       <div className="last-container">
-        <span>Current Chapter</span>
-        <h3>{currentChapter}</h3>
```
Comment (FrederickMih) [inline src/components/Book.js:29]: - [x] Kindly make sure that the font weight is exactly the same as in Zeplin.

File: src/components/Form.js [hunk 3]
```diff
@@ -26,11 +26,12 @@ export default function Form() {
...
   };
   return (
-    <div>
+    <div className="form-group">
+      <h3 className="form-title">ADD NEW BOOK</h3>
       <form className="form-body" onSubmit={handleSubmit}>
         <input type="text" name="autor" placeholder="Autor" onChange={(e) => setAuthor(e.target.value)} />
         <input type="text" name="book" placeholder="Book" onChange={(e) => setBook(e.target.value)} />
-        <button type="submit">Add Book</button>
+        <button className="submit" type="submit">ADD BOOK</button>
```
Comment (FrederickMih) [inline src/components/Form.js:34]: - [x] Also, align this button with the main container in order to match the original design.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @digia3d , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :green_apple: 

- GitHub workflow has been followed :+1: 
- PR has a title and description :+1: 
- No Linter error detected :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @digia3d, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
- [x] Next time, and for future Pull Requests, please kindly add a more descriptive PR title.
See a demo below:
pull_request_desc
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**